pkg/core/blockchain_test.go

```diff
@@ -144,6 +161,7 @@ func TestGetTransaction(t *testing.T) {
 	bc := newTestChain(t)
 	// Turn verification off, because these blocks are really from some other chain
 	// and can't be verified, but we don't care about that in this test.
+	// FIXME this needs to be fixed
```

Comment:
Do we really care? If yes, then there should be an issue, not a comment. But I doubt so.

Reply:
I see no value in using decoded block instead of generating them on-the-fly. We just check for the presence of a single transaction.

Reply:
Well, from `GetTransaction` testing perspective it might be true. It's just that at the moment we have some very particular transaction and we use knowledge of its internals in the test. Maybe it's not the useful, though.